src/Robo/Commands/Vm/VmCommand.php

```diff
@@ -277,6 +277,10 @@ protected function setBaseBox($config) {
     switch ($base_box) {
       case 'beet/box':
         $config->set('workspace', '/beetbox/workspace/{{ php_version }}');
+        $config->set('extra_packages', [
```

Comment:
Could this and the other change in this file be refactored to be more DRY? Maybe something like
```
$config->set('extra_packages', $this->getExtraPackages());

protected function getExtraPackages() {
   return [
          'patchutils',
          'sqlite',
        ];
}
```

Reply:
Actually, better is probably to allow packages to be added in the `box/config.yml` instead of hardcoding them in.

Reply:
@ba66e77 I am following the pattern used for `installed_extras` -- not sure why this should be any different. Thoughts? I think an overall refactor should be done in a separate issue if that is a change that is requested.

(I included `sqlite` since that is currently installed by default by DrupalVM we should maintain that.)

Reply:
@ba66e77 @danepowell ping.

Reply:
I agree that an overall refactor to make this configurable would be good. But that should probably be a follow-up issue. Since this PR follows the existing pattern, I'm okay with it.

Reply:
Created https://github.com/acquia/blt/issues/3252 for the followup.